components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -760,27 +778,31 @@
                             if (propertyName != null && propertyName.trim().length() > 0) {
 
                         %>
-                        <td class="leftCol-med" width="50%" style="text-align:left;" id="<%=name%>"><%=propertyName%>
+                        <td class="leftCol-med" width="50%" style="text-align:left;"
+                        id="<%=Encode.forHtmlAttribute(name)%>"><%=Encode.forHtml(propertyName)%>
```

Comment:
variable name is hardcoded and does not contain special characters. No need to encode that variable